Creates the July Newsletter (#181)
readthedocs/ethicalads.io: content/posts/newsletter-july-2021.md

Title: EthicalAds Newsletter - July 2021

> Date: August 11, 2021

Reviewer: ```suggestion
Date: August 18, 2021
```

> description: Our July 2021 ads update, including stats and commentary, on our progress as we build EthicalAds

Reviewer 2: ```suggestion
description: Our July 2021 ads update, including stats and commentary on our progress as we build EthicalAds
```

tags: newsletter, community
authors: Ra Cohen
image: /images/posts/2021-july-newsletter.jpg
image_credit: <span>Photo by <a href="https://unsplash.com/@ingmarr?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Ingmar</a> on <a href="https://unsplash.com/t/nature?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Unsplash</a></span>

## New features

These are the major features we released in the last month:

* In our ad server, we added ```noopener``` to our ad links as a security precaution.

> * Our ad server now raises a warning on link redirects and errors.

Reviewer 2: ```suggestion
* Our ad server now raises a warning on link redirects and errors when advertisers are creating an ad, to make sure it isn't an error in the URL. 
```

* Also in our ad server, we created actions to allow our staff to handle
  the creation and customization of new advertising flights.
* On our website, we created our first iteration of an **ethical** marketing funnel
  by implementing a transparent, human-readable first-person cookie,
  running the open-source, privacy-friendly [Plausible analytics](https://plausible.io/vs-google-analytics),
  and creating our first ad-specific landing page for [prospective Python publishers](https://www.ethicalads.io/publishers/topics/python/).

>   Lookout for a blog post on how we did it, coming out later this month!

Reviewer 2: ```suggestion
  Lookout for a blog post on how we did it, coming out soon!
```

Never promise specific dates, unless we've already done it :)

* Also on our website, we launched our [Publisher Referral Offer](https://www.ethicalads.io/blog/2021/07/ethicalads-publisher-referral-offer-july-2021/).

> You can always see our latest server updates in our [ethical-ad-server changelog](https://ethical-ad-server.readthedocs.io/en/latest/developer/changelog.html).

Reviewer 2: ```suggestion
You can always see our latest server updates in our [ethical-ad-server changelog](https://ethical-ad-server.readthedocs.io/en/latest/developer/changelog.html) and [ethical-ad-client changelog](https://ethical-ad-client.readthedocs.io/en/latest/changelog.html).
```

## July advertising stats

> July's performance was complicated.

Reviewer 2: This feels a bit incomplete. We should probably move up the copy from below the stats to here, to explain a bit more.

Author: ```suggestion
In July we removed a number of under-performing publishers and onboardeded some
new publishers, though we continue to [look for more](https://www.ethicalads.io/blog/2021/07/ethicalads-publisher-referral-offer-july-2021/)!
We also added some globally-targeted ad-buys which boosted our total number of paid views. 
```

In July 2021:

* We generated **$23,981.08** overall for our publishers,
  a slight decrease from June.
  Of this, $12,966.47 was to third-party publishers,
  a slight increase from June.

>   The rest, as usual, went to Read the Docs.

Reviewer 2: I'd like to drop this split, and just report the total number. I don't think this provides any value anymore, so we should just report a single number.

Reviewer: I'm fine with that

Author: ```suggestion
* We generated **$23,981.08** for our publishers,
  a slight decrease from June.
```

* We had **105 publishers** on our network with at least one paid ad impression,
  a slight decrease from June.
* We had **16,257,531** paid ad views across the world,
  a sizable increase from June's ~14.6 million views.

In an effort to maintain the high quality of our network,
we removed some under-performing publishers and onboarded some
new publishers, but we are still looking for more!
This accounts for the slight decrease in the number of publishers

> and the slight increase in the number of paid views.

Reviewer 2: I believe the increase in ad views is because of a global buy, not more publishers.

Reviewer: This is correct.

## Tricky bits

We are still verging on max capacity with regards to ad inventory specifically in the US and EU.
We focused our efforts this month are marketing in order to attract new publishers.
We rolled out our new marketing funnel and were even featured on
[HackerNews](https://news.ycombinator.com/item?id=27987650)
[twice](https://news.ycombinator.com/item?id=27975685).
Unfortunately, determining the success of these new endeavors
takes time as we don't want to change too many variables at once,
lest we not know which avenues had the greatest impact.

## Upcoming features

The major features in our upcoming roadmap:

* Implementing a view time metric for our ads to ensure a fair system for counting valid views

>   as well as provide a metric for publisher quality.

Reviewer 2: ❤️
